README.md

```diff
@@ -0,0 +1,77 @@
+# SkyPortal Mobile
+
+This is the mobile version of the [SkyPortal](https://github.com/skyportal/skyportal) platform. It is built with Ionic
+React and is available for both iOS and Android.
+
+## Development requirements and setup
+
+### Node.js and npm
+
+To run the app locally, you will need to have Node.js and npm installed. You can install them
+using [nvm](https://github.com/nvm-sh/nvm) or just install a recent version from the [node website](https://nodejs.org/).
+
+### Android Studio
+
+To be able to build the Android version, you will need to have Android Studio installed with the Android SDK and the
+Android SDK Platform-Tools. You can follow the instructions [here](https://developer.android.com/studio/install) to
+install Android Studio. Then, open Android Studio, go to **Tools -> SDK Manager** and install the latest stable Android
```

Comment:
"latest stable" could be a problem. I believe that `android/variables.gradle` pins a specific version? Worth double checking.

Reply:
It sets a minimum version. The latest stable will be above it for sure

Reply:
Ok! Reading it I though it was setting a minimum version + a target version for something else, but you might be right. I'll trust your judgement here.